### PR TITLE
Added time complexity analysis

### PR DESCRIPTION
**Pull request by Lay-Sin**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -50,8 +52,10 @@ def quick_sort(array, low, high):
 x = 0
 
 # Sort words
+a = timeit.default_timer()
 quick_sort(wordlist, 0, len(wordlist) - 1)
 x = x + 3
+print(a)
```
Comment (Michelle-Lohwt) [inline main.py]: 1. Shouldn't `a` be before the `x = x + 3`
2. Should we include the space complexity within the function? My concern is the `timeit` will also include the space complexity when it runs the function.

NOTE: Feel free to correct me if you think I am wrong, I am open for discussion🙌

**Review by Michelle-Lohwt — COMMENTED** [review]
Please check the comments

**Comment by Lay-Sin** [comment]
Actually if timeit function is used and the result is reasonable, then I plan to delete all the counter things. So, actually that x = x + 3 will be deleted.

**Comment by Michelle-Lohwt** [comment]
Sure, no problem, you can change it, then I'll review and approve the pull request (PR)

**Comment by Lay-Sin** [comment]
Probably you all can test out the timeit function at your machine first. If all goes smoothly and the result is reasonable, please inform me and I will delete the counter. Thank you.

**Comment by Michelle-Lohwt** [comment]
Will do

**Comment by Michelle-Lohwt** [comment]
The `timeit` will do ya, no need the counter @Lay-Sin

**Comment by Lay-Sin** [comment]
Alright. Later will update it.

**Comment by Lay-Sin** [comment]
@Michelle-Lohwt Done remove the counter. Can have a look.

**Review by Michelle-Lohwt — APPROVED** [review]

**Review by ziqiang2344 — APPROVED** [review]

**Pull request merged**